modules/python/init.zsh

```diff
@@ -92,22 +92,31 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
   # Disable the virtualenv prompt.
   VIRTUAL_ENV_DISABLE_PROMPT=1
 
-  # Enable 'virtualenv' with 'pyenv'.
-  if (( $+commands[pyenv] )) && \
-     pyenv commands | command grep -q virtualenv-init
-  then
-    eval "$(pyenv virtualenv-init -)"
-    # Optionall activate 'virtualenvwrapper' with 'pyenv' is available.
-    if (( $#commands[(i)pyenv-virtualenvwrapper(_lazy|)] )); then
-      pyenv "${${(@O)commands[(I)pyenv-virtualenvwrapper(_lazy|)]}[1]#pyenv-}"
+  if (( $+commands[pyenv] )); then
+    # Grab sorted array of available 'pyenv' plugins by probing 'pyenv'
```

Comment:
I'm not sure this comment matches the code pre se.  It looks like the code is returning all pyenv commands that start with "virtualenv", and not all pyenv plugins, correct?